test/demo/activation-keys.yml

```diff
@@ -0,0 +1,17 @@
+---
+- hosts: localhost
+  tasks:
+  - name: "Create katello client activation key"
+    katello_activation_key:
+      username: "admin"
```

Comment:
shall we split out username/password/server_url/organization into an fixture file, then it would be easier to make demos in a slightly different env?